### PR TITLE
docs(discovery): add eureka and nacos

### PR DESCRIPTION
**Pull request by okaybase**
### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/zh/latest/discovery/eureka.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+Apache APISIX 支持使用 Eureka 做服务发现。
```
Comment (tokers) [inline docs/zh/latest/discovery/eureka.md:24]: What about adding a hyperlink for Eureka here?

Comment (okaybase) [reply docs/zh/latest/discovery/eureka.md]: okay~ done~ thanks~ @tokers

Comment (tokers) [reply docs/zh/latest/discovery/eureka.md]: Actually, I mean add a hyperlink about what is Eureka.

Comment (okaybase) [reply docs/zh/latest/discovery/eureka.md]: okay~ got~ done~ thanks~ @tokers

**Review by tokers — APPROVED** [review]

**Review by tzssangglass — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Pull request merged**